### ISSUE TITLE
Compilation from latest sox source?

### ISSUE THREAD

**Issue by God-damnit-all** [turbulentie/sox-dsd-win#1]
Thanks for compiling this, but in the README you indicate you're using the source from here: https://github.com/mansr/sox

That hasn't been the main repo in a long time. The developer now uses sourceforge exclusively: https://sourceforge.net/p/sox/code/ci/master/tree/

You can see that a lot has been done since 14.4.2: https://sourceforge.net/p/sox/code/commit_browser

In particular I noticed fixes to MP3 ID3 tagging.

Would it be possible to get a compilation for the latest head commit of the sourceforge repo? Even better would be a GitHub Actions workflow, but beggars can't be choosers.

Even if you don't have the time for this, I appreciate uploading your 14.4.2 DSD compilation nonetheless.

**Comment by turbulentie** [comment]
Yes. To be honest i had idea to update binaries. I will recompile it with latest sources. And thanks for recommendations.
